screens/Registration/InformationScreen/InformationScreen.tsx

```diff
@@ -30,8 +30,32 @@ const InformationScreen: React.FC = () => {
   const [dateModal, setDateModal] = useState(false)
   const [hasBeenTouched, setHasBeenTouched] = useState(false)
   const useNav = useNavigation()
+  const [keyboardVisible, setKeyboardVisible] = useState(false)
 
-  const { firstName, setFirstName, lastName, setLastName, date, setDate, location, setLocation } = useContext(RegistrationContext)
+  useEffect(() => {
+    const showSubscription = Keyboard.addListener("keyboardDidShow", () => {
+      setKeyboardVisible(true)
+    })
+    const hideSubscription = Keyboard.addListener("keyboardDidHide", () => {
+      setKeyboardVisible(false)
+    })
+
+    return () => {
+      showSubscription.remove()
+      hideSubscription.remove()
+    }
+  }, [])
```

Comment:
This seems to be recurring, why don't you create a custom hook called `useKeyboard` to access this more easily? I don't think you've ever made custom hook but it's really easy can help you out if needed!

Reply:
Done!